web/server.js

```diff
@@ -17,14 +16,14 @@ var passTest = new PassTest({
   allowPassphrases: true
 });
 
-module.exports = function(options) {
+module.exports = async function server(options) {
   var serverConfig = {
     debug: options.debug,
-    connections: {
-      routes: {
-        security: true
-      }
-    }
+    routes: {
```

Comment:
changes in the API so now we have to pass the host/port directly to the constructor